examples/basic/index.html

```diff
@@ -153,15 +151,23 @@
 }
 
 function onResize() {
-  console.log('Resizing to %s x %s.', window.innerWidth, window.innerHeight);
-  effect.setSize(window.innerWidth, window.innerHeight);
-  camera.aspect = window.innerWidth / window.innerHeight;
-  camera.updateProjectionMatrix();
+  // The delay ensures the browser has a chance to layout
+  // the page and update the clientWidth/clientHeight.
+  if(!this.resizeDelay) {
+    this.resizeDelay = setTimeout(() => {
```

Comment:
:+1: on using a bound function rather than an arrow function -- this will just fail to parse on those browsers (as much as I love arrow functions)